src/Sylius/Bundle/ApiBundle/Swagger/PathHiderDocumentationNormalizer.php

```diff
@@ -31,6 +32,7 @@ public function supportsNormalization($data, $format = null): bool
     public function normalize($object, $format = null, array $context = [])
     {
         $docs = $this->decoratedNormalizer->normalize($object, $format, $context);
+        Assert::isArray($docs);
         $paths = (array) $docs['paths'];
```

Comment:
I didn't check for `$docs['paths']` contents, but it looks like typecast handling missing key and not for `'paths'` to be typecasted to array.

If I'm mistaken, then second suggestion.
```suggestion
        $paths = $docs['paths'] ?? [];
```
```suggestion
        $paths = (array) $docs['paths'] ?? [];
```

![image](https://github.com/Sylius/Sylius/assets/870747/3df81997-cec5-4225-8f53-4b5436ea2806)

https://onlinephp.io/c/03539